src/main/scala/bozzy/controllers/FilterPaneController.scala

```diff
@@ -19,9 +21,21 @@ class FilterPaneController (private val filterLabel: Label,
                             private val chordCount: Spinner[String],
                             private val dictionary_box: ChoiceBox[String],
                             private val wordCount: Spinner[String],
-                            private val collisions_checkbox: CheckBox) {
+                            private val collisions_checkbox: CheckBox,
+                            private val hide_duplicates_checkbox: CheckBox) {
 
-  dictionary_box.items = StenoDictionary.openDictionaryNames
+  dictionary_box.items = new SortedBuffer[String](MainDictionary.dictionaryFilterChoices) {
+    comparator = new StringOrdering {
+      override def compare(x: String, y: String) =
```

Comment:
This could work too, if you prefer:
        (x, y) match {
          case ("Any", _) => -1
          case ("_", "Any") => 1
          case _ => super.compare(x,y)
        }